Cargo.toml

```diff
@@ -34,6 +34,7 @@ crossbeam = { version = "0.8.4" }
 dirs = { version = "5.0.0" }
```

Comment:
We can remove `dirs` in a future release, but it stays around for now for backwards compatibility.